test/integration/targets/cs_prepare/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
There shouldn't be any need for `cs_prepare` with `ansible-test`. It takes care of the requirements when running with any of the options: `--tox`, `--docker` or `--requirements`, as well as configuring the environment and fetching the API keys from the simulator.